### DISCUSSION TITLE
[RFC] Move the Field Guide out of the classifier?

### DISCUSSION THREAD

**Discussion by eatyourgreens** [zooniverse/front-end-monorepo#2424] [Ideas]
**Package**
lib-classifier

**Feature or Issue Description**
The field guide is a project resource that gets fetched each time the classifier mounts, but we only really need to get it once, when the project loads. It also needs to be available outside the classifier. For instance, moderators refer to the Field Guide from Talk when giving advice about how to classify.

I'm wondering if the field guide store should be moved out of the classifier, and the field guide itself should be a shared component? Or should it be something that can be linked to, for easier referencing from Talk posts?

**Comment by rogerhutchings** [comment]
It could be its own package, which exports its components _and_ a route object describing how to link to various states. It'd then be the job of the consuming package to hook up those routes.

cc #650, where I suggested breaking out the LCV into its own package(s)

**Comment by eatyourgreens** [comment]
For an example of a Field Guide with linkable URLs, see OWD (which links to these URLs from the transcription tools.):
https://www.operationwardiary.org/#/guide/diary/diaryDate
https://www.operationwardiary.org/#/guide/diary/person
I don't know if we ever did that for any other projects.

**Comment by srallen** [comment]
The Wildcam Gorongosa field guide is linked to from its classifier. https://www.wildcamgorongosa.org/#/field-guide
https://www.wildcamgorongosa.org/#/field-guide/baboon

**Comment by eatyourgreens** [comment]
Excellent. I think it was also camera survey projects that asked to have the Field Guide available from Talk. Or, at least, they yelled when we removed it from Talk in PFE without asking the moderators first.

**Comment by beckyrother** [comment]
This is great. Would be nice to open in a smaller window in front of the classifier so it doesn't detract from the classification experience and matches the original design intent as closely as possible.

Separately, the back button should return to the scroll position. Talk comment: https://www.zooniverse.org/projects/webster-institute/mapping-historic-skies/talk/2781/1217524

**Comment by srallen** [comment]
We have had a suggestion come in about making the field guide print to pdf-able: https://zooniverse.freshdesk.com/a/tickets/3024

**Comment by beckyrother** [comment]
That could be interesting. Could also be a result of not being able to keep the field guide in a separate window while classifying – might be worth revisiting this ask after the popup is implemented.

**Comment by snblickhan** [comment]
Commenting here so I get notified of activity, but also to note that the FG staying open during classification is a really common feature request for Humanities projects that want to include transcription style guides, etc.

**Comment by eatyourgreens** [comment]
Some projects are uploading guides as PDFs, so that you can refer to them while classifying. https://www.zooniverse.org/talk/18/2111021?comment=3461404
